prowspecs/specs.go

```diff
@@ -71,6 +71,26 @@ var knownBranches map[string]BranchSpec = map[string]BranchSpec{
 
 		skipTrivy: true,
 	},
+	"release-1.11": {
+		prowContext: &prowgen.ProwContext{
+			Branch: "release-1.11",
+
+			// Freeze test images used.
+			Image: "eu.gcr.io/jetstack-build-infra-images/bazelbuild:20220830-c65cd19-4.2.1",
+
+			// NB: we don't use a presubmit dashboard outside of "master", currently
+			PresubmitDashboard: false,
+			PeriodicDashboard:  true,
+
+			Org:  "cert-manager",
+			Repo: "cert-manager",
+		},
+
+		primaryKubernetesVersion: "1.25",
+		otherKubernetesVersions:  []string{"1.20", "1.21", "1.22", "1.23", "1.24"},
+
+		skipTrivy: true,
```

Comment:
@SgtCoDFish Should we still skipTrivy here?

In https://github.com/cert-manager/release/pull/105#discussion_r991411128 @irbekrm asked:
> Question: why do we skip trivy tests here- from [this comment](https://github.com/cert-manager/release/blob/bfa6548a34fd6bc9441fbe60e7805f988e187c8b/prowspecs/specs.go#L134) it seems like they are skipped for versions where the trivy change wasn't backported? I assume we might want to run them against an already released version if it is the case that an updated trivy database can find new vulnerabilities in an already released version.

At the time @jahrlin said the trivy tests were broken, but https://github.com/cert-manager/release/pull/107 may have fixed that.

Reply:
> At the time @jahrlin said the trivy tests were broken, but https://github.com/cert-manager/release/pull/107 may have fixed that.

They are fixed, running daily and passing now! E.g. [the controller](https://testgrid.k8s.io/cert-manager-periodics-master#ci-cert-manager-master-trivy-test-controller).

> Should we still skipTrivy here?

I think there's value in running them; it would be a shame to release a version which doesn't have the latest vulns fixed and trivy could help us spot that. That said, my longer comment which I'll write in a sec may make this a moot point!